ex03.ex

```diff
@@ -77,7 +80,11 @@ defmodule Ex03 do
 
   """
 
-  def list_contains . .. "your code"
+
+  def list_contains([ head | tail ], num) when head == num, do: true
```

Comment:
## -1

No need for a test when we can use patterns:

~~~ elixir
def list_contains([ num | tail ], num), do: true
~~~